plugins/python/python_plugin.c

```diff
@@ -1813,13 +1827,22 @@ void uwsgi_python_resume(struct wsgi_request *wsgi_req) {
 	PyThreadState *tstate = PyThreadState_GET();
 	PyGILState_Release(pgst);
 
-	if (wsgi_req) {
```

Comment:
i guess this was unwanted

Reply:
Fixed. Oops, I made a mistake when I adapted my patch from uwsgi-2.0.20 to the master branch.